Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -142,10 +147,17 @@ internal final class CommentsViewController: UITableViewController {
         $0 ? self?.refreshControl?.beginRefreshing() : self?.refreshControl?.endRefreshing()
       }
 
-    self.viewModel.outputs.showLoadingIndicatorInFooterView
+    self.viewModel.outputs.configureFooterViewWithState
       .observeForUI()
-      .observeValues { [weak self] shouldShow in
-        self?.footerView.configureWith(value: shouldShow)
+      .observeValues { [weak self] state in
+        self?.footerView.configureWith(value: state)
+
+        // Force tableFooterView to layout
```

Comment:
Not pretty but wasn't really successful getting the footer to find the correct size any other way 🤷‍♂️ Open to suggestions.

Reply:
I think an alternative might be to use entirely different footer views and switch them out but if we go that route I'll do it in a separate PR as an iteration on this.